lib/fieldhand/record.rb

```diff
@@ -45,6 +45,13 @@ def sets
       header.sets
     end
 
+    # Return this whole item as a string
+    #
+    # A record can be in any format, Fieldhand does not attempt to parse it but leaves that to the user.
+    def dump
+      Ox.dump(element)
```

Comment:
One thing we found we had to do was explicitly set Ox's encoding to `utf-8`, could you see if it can successfully dump XML with Unicode characters in it without that?